github custom properties (#2209)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/git/github/examples/resource-mapping-examples.md

<RepositoryTagReleaseAppConfig/>

## Map repositories and repository custom properties

> In the following example you will ingest your GitHub repositories and their custom properties to Port, you may use the following Port blueprint definitions `port-app-config.yml`:

Reviewer: ```suggestion
The following example shows how to ingest your GitHub repositories and their custom properties to Port.  
You can use the following blueprint definitions and `port-app-config.yml`:
```
